packages/cascara/src/modules/ModuleError/ModuleError.js

```diff
@@ -4,7 +4,7 @@ import styles from '../DataModule.module.scss';
 
 const propTypes = {
   moduleName: pt.string.isRequired,
-  moduleOptions: pt.array.isRequired,
+  moduleOptions: pt.arrayOf(pt.shape()).isRequired,
```

Comment:
Use `option` when it is valid. 

I wish the lint rule had a way to not let an empty shape pass.

Reply:
Could you please clarify this request?

Reply:
If using `pt.object` is valid, we should use that as the type, instead of just using `pt.shape({})` with an empty value.

My comment was just that I wish the forbidden proptypes rule had a way to catch the empty `pt.shape({})` and make it an error. The only request on these is to change them to `pt.object` with a eslint comment to disable the rule explaining why we are doing it.